packages/web-components/src/components/va-alert/va-alert.tsx

```diff
@@ -153,12 +157,16 @@ export class VaAlert {
     if (!visible) return <div aria-live="polite" />;
 
     return (
-      <Host>
+      <Host role="presentation">
         <div role={role} aria-live={ariaLive} class={classes}>
           <i aria-hidden="true" role="img"></i>
-          <div class="body" onClick={this.handleAlertBodyClick.bind(this)}>
+          <div class="body">
             {!backgroundOnly && <slot name="headline"></slot>}
-            <slot></slot>
+            <div id="body-click" role="presentation">
+              <div>
```

Comment:
In the NVDA link, there is this snippet:

```html
<div> <!-- for display:block to create consistency of reading in NVDA -->
```

If the extra div is just there for setting the display, could we have css that's like:

```css

/** For NVDA reading **/
#body-click {
  display: block;
}
```

And not need the `<div>` on 166?

This isn't a huge deal, but I can see this kind of thing easily getting removed in the future if someone comes in here and sees a cascade of divs without any particular styling applied to them. They might remove it as a "refactoring cleanup" and re-introduce the problem. Including CSS in the solution might make things more explicit.

Reply:
This isn't working right now. It still says clickable no matter if I use addEventListener, onClick, and role. This is looking like a [React 17 issue](https://github.com/facebook/react/issues/20895) with screen readers. This "clickable" problem is happening on our other components including React components in Storybook.

I'll try the @Listen.

Reply:
Without `role="presentation"` on the React root it doesn't matter what I tried: `onClick` on div with class of `body` or separate div, `addEventListener` and `@Listen` didn't prevent NVDA from announcing va-alert as clickable.

With `role="presentation"` on the React root AND `role="presentation"` on the div with the onClick, va-alert is no longer announced as clickable.